src/components/forms/SearchBar/SearchBar.stories.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react/types-6-0';
+import { action } from '@storybook/addon-actions';
+
+import SearchBar from './SearchBar';
+import { SearchBarProps } from './SearchBar.types';
+
+export default {
+  title: 'components/forms/SearchBar',
+  component: SearchBar,
+} as Meta;
+
+const delay = (ms) => new Promise((res) => setTimeout(res, ms));
+
+const mockOnSearch = async (query) => {
```

Comment:
Please move this mock into `Datatable/mocks/search.ts`

Reply:
fixed in last commit: moved to SearchBar/mocks.ts